discovery_engine/test/discovery_engine/utils/utils.dart

```diff
@@ -188,6 +188,8 @@ final mockConfig = Configuration(
   applicationDirectoryPath: './',
   feedMarkets: {const FeedMarket(countryCode: 'DE', langCode: 'de')},
   manifest: Manifest.fromJson(goodJson),
+  headlinesProviderPath: '/newscatcher/v1/latest-headlines',
```

Comment:
Are this path in prod?

Reply:
[Does not look like it](https://github.com/xaynetwork/xayn_infrastructure/search?q=latest-headlines)

Reply:
I've opened TY-2995 to track this.